scummvm.appdata.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please submit it upstream.

Reply:
[Done](https://github.com/scummvm/scummvm/pull/1213). But I still have to use the local version because of the information about sandboxing that I added.